### PR TITLE
Add Web3ID verifier in typescript that can be extended

### LINKED ISSUE #137: Create Web3Id Verifier in TS
- [x] Extend web-sdk with necessary functionality
  - [X] Verify web3-id presentation (rust-bindings)
  - [x] Get public data of all credentials of presentation (see rust-sdk web3id implementation)
- [x] Implement web3-id verifier example server in typescript

### PR DESCRIPTION
**Pull request by Bargsteen**
## Purpose

Add Web3ID verifier in typescript that can be extended.

Depends on changes introduced in the JS SDK in https://github.com/Concordium/concordium-node-sdk-js/pull/357.

Closes #137

## Changes

- Add a new verifier service written in typescript

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: services/web3id-verifier-ts/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Concordium dApp template
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```
Comment (limemloh) [inline services/web3id-verifier-ts/README.md:3]: **Request change:** The top description in the readme should probably be updated.
Documenting the purpose of this service and the endpoint exposed would be nice.

**Review by limemloh — APPROVED** [review]
Looks good to me, only a comment related to the readme

File: services/web3id-verifier-ts/README.md (hunk 1 above)
Comment (soerenbf) [reply services/web3id-verifier-ts/README.md]: Done, thanks for bringing this up!

File: services/web3id-verifier-ts/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "scripts": {
+        "build": "tsc",
+        "start": "tsx src/main.ts",
+        "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+        "lint-fix": "yarn lint --fix",
+        "fmt": "prettier -w .",
+        "fmt-check": "prettier --check ."
+    },
+    "dependencies": {
+        "@concordium/web-sdk": "^7.4.0-alpha.4",
```
Comment (soerenbf) [inline services/web3id-verifier-ts/package.json:16]: Use official release when available

**Pull request merged**